training_system/app/controllers/trainer/courses_controller.rb

```diff
@@ -70,4 +73,11 @@ def load_subjects course
     @subjects = course.subjects.page(params[:page])
                       .per(Settings.courses.per_page)
   end
+
+  def trainee_not_course
+    trainee = @course.users.trainee.ids
+    @trainees_not_course = User.trainee.where.not(id: trainee)
```

Comment:
`.where.not(id: trainee)` -> viết scope cho code này nhé :v sau có khi cần tái sử dụng